### PR TITLE
Update 'Manage Assets' guidance

### PR DESCRIPTION
**Pull request by brucebolt**
Several steps have changed since moving to Kubernetes. Therefore updating the guidance to reflect these changes.

Also makes some changes to other parts of the documentation to make the steps clearer.

[Trello card](https://trello.com/c/4J8IJaa1)

### CODE REVIEW

File: source/manual/manage-assets.html.md.erb [hunk 1]
```diff
@@ -60,71 +80,108 @@ not provide the facility for editors to upload assets such as images and PDFs.
...
-```sh
-$ gds govuk connect scp-push -e production aws/backend:1 my_file.jpg /tmp
-```
+> If the pod terminates at any point during this process (e.g. a deployment occurs),
+> you will need to start again as the `/tmp` directory is specific to the pod.
 
-Then SSH to the same machine and run the upload command:
+1. Get the name of the asset-manager pod (you need the app pod, which will be named
+   like `asset-manager-abc1234-abc123` and does not contain any extra words, such as
+   `freshclam`, `pss` or `worker`):
```
Comment (jkempster34) [inline source/manual/manage-assets.html.md.erb:91]: We could put this information in the grep, but it is perhaps less approachable 

```
kubectl get pods --all-namespaces | grep asset-manager | grep -v -e freshclam -e pss -e worker
```

**Review by jkempster34 — COMMENTED** [review]
All looks good to me with a few suggestions. I haven't actually ran the commands though!

File: source/manual/manage-assets.html.md.erb [hunk 2]
```diff
@@ -11,24 +11,36 @@ parent: "/manual.html"
 If you need to remove an asset manually from `assets.publishing.service.gov.uk`,
 follow these steps:
 
+> Ideally, the deletion should be done in the publishing application that originally
+> uploaded the file (e.g. Whitehall). Deleting an asset from Asset Manager without
+> removing the record from the publishing application could cause problems if the
+> asset is attempted to be re-published again in the future.
+
+1. Get the asset ID. This can be obtained from the URL, e.g. the asset ID for
```
[the remark this answers is not in the document]
Comment (brucebolt) [reply source/manual/manage-assets.html.md.erb:19]: I believe all Whitehall Assets are now served from URLs that contain their database ID and we're no longer using the legacy URL path.

File: source/manual/manage-assets.html.md.erb [hunk 3]
```diff
@@ -11,24 +11,36 @@ parent: "/manual.html"
 If you need to remove an asset manually from `assets.publishing.service.gov.uk`,
 follow these steps:
 
+> Ideally, the deletion should be done in the publishing application that originally
+> uploaded the file (e.g. Whitehall). Deleting an asset from Asset Manager without
+> removing the record from the publishing application could cause problems if the
+> asset is attempted to be re-published again in the future.
```
[the remark this answers is not in the document]
Comment (brucebolt) [reply source/manual/manage-assets.html.md.erb:17]: Good idea.  I've added similar wording in all three places where I've suggested the change should ideally be made in the publishing app.

File: source/manual/manage-assets.html.md.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (jkempster34) [reply source/manual/manage-assets.html.md.erb:22]: IDK much about these, but this one doesn't have an ID https://assets.publishing.service.gov.uk/government/uploads/uploaded/hmrc/realtimepayetools-update-v23.xm

[the remark this answers is not in the document]
Comment (brucebolt) [reply source/manual/manage-assets.html.md.erb:22]: 😬

Ooh, didn't notice those hadn't been migrated.  I'll add another step for these.

File: source/manual/manage-assets.html.md.erb (hunk 1 above)
Comment (brucebolt) [reply source/manual/manage-assets.html.md.erb:113]: I did consider this, but decided it's not a definitive list as it's possible pods with other names could be added in time.

Comment (jkempster34) [reply source/manual/manage-assets.html.md.erb:113]: Agreed!

**Review by jkempster34 — APPROVED** [review]
👍

**Pull request merged**